File: OSCommon/org/optimizationservices/oscommon/datastructure/osbenchmark/BenchmarkHeader.cs
using System;

namespace org.optimizationservices.oscommon.datastructure.osbenchmark{
	/// <summary>
	/// A data structure class that corresponds to an xml element in the OSbL schema. 
	/// @author Jun Ma	
	/// @version 1.0, 09/01/2005
	/// @since OS 1.0
	/// @copyright (c) 2005
	/// </summary>
	public class BenchmarkHeader{
		
		public string serviceURI = "";

		public string serviceName = "";

		public DateTime time = new DateTime(1970, 1, 1, 0, 0, 0);

		public string message = "";

	}//class BenchmarkHeader
}//namespace

File: OSCommon/src/org/optimizationservices/oscommon/datastructure/osbenchmark/BenchmarkHeader.java
/**
 * @(#)BenchmarkHeader 1.0 03/14/2004
 *
 * Copyright (c) 2004
 */
package org.optimizationservices.oscommon.datastructure.osbenchmark;

import java.util.GregorianCalendar;

/**
*
* <p>A data structure class that corresponds to an xml element in the OSbL schema.  
*  
* </p>
* @author Robert Fourer, Jun Ma, Kipp Martin
* @version 1.0, 03/14/2004
* @since OS 1.0
*/
public class BenchmarkHeader {
	
	public String serviceURI = "";

	public String serviceName = "";

	public GregorianCalendar time = null;
	
	public String message = "";

}//BenchmarkHeader
